src/test/java/com/epam/tc/hw3/AbstractTest.java

```diff
@@ -0,0 +1,54 @@
+package com.epam.tc.hw3;
+
+import static utils.Config.getUserNameFromProperties;
+import static utils.Config.getUserPasswordFromProperties;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
+
+
+public class AbstractTest {
+
+    static String LOGIN_ROMAN_IOVLEV = "ROMAN IOVLEV";
+    public static WebDriver webDriver;
+    public static String URL_HOME_PAGE = "https://jdi-testing.github.io/jdi-light/index.html";
+    static String HOME_PAGE = "Home Page";
```

Comment:
URL_HOME_PAGE I think it's part of the page. So, it can be moved to a page class